contrib/awssqs/PROJECT

```diff
@@ -0,0 +1,3 @@
+version: "1"
```

Comment:
where is this message read ?

Reply:
ahm... file :-)

Reply:
I took this from gcppubsub in an attempt to keep things consistent, but I admit I don't know the purpose of the file.

Reply:
I took this from gcppubsub in an attempt to keep things consistent, but I admit I don't know the purpose of the file.

Reply:
It's used by kubebuilder to identify the repo root I think. I don't think it needs to be copied to contrib.